src/main/resources/static/js/subscription.js

```diff
@@ -414,6 +414,10 @@ jQuery(document).ready(function () {
             }
         }
     });
+
+    $("#sidenavCollapse").click(function() {
+		table.ajax.reload();
```

Comment:
Does this mean that Datatable is reloaded and it re-retrieve data from backend, e.g. it makes a Get all subscriptions request everytime sidebar is collapsed?

Reply:
Apparently the function was unable to detect dataset changes but I've now changed it with a better method.